app/helper/supplier_items_helper.rb

```diff
@@ -6,7 +6,7 @@ def add_categories_to_item(categories, item)
   end
 
   def get_all_categories
-    @categories = Category.all
+    @categories = Category.all.uniq
```

Comment:
This works, but should we add unique category validations to the model and rebuild the database if it is not there already?